src/Rule/Nested.php

```diff
@@ -342,11 +304,10 @@ public function getNoPropertyPathMessage(): string
      * @throws InvalidArgumentException When rules' source has wrong type.
      * @throws InvalidArgumentException When source contains items that are not rules.
      */
-    private function prepareRules(iterable|object|string|null $source): void
+    private function setRules(iterable|object|string|null $source): void
```

Comment:
Maybe rename to something like `initRules`? For me, `set` prefix is more associated with public setters.

Reply:
Have a better idea for naming in #568.